index.html

```diff
@@ -1634,7 +1714,7 @@ <h1>DID Methods</h1>
 
     <p>
 This table summarizes the DID method specifications currently in development.
-The links will be updated as subsequent Implementer’s Drafts are produced.
+The links will be updated as subsequent Implementer’s Drafts are produced. <a href="https://ipfs.io/ipfs/QmZ5eDK3k12hYWd2EPKJNkPMLokZ8gXuPASUA7D4B3YVy2" target="_blank">CDDL definitiion of DID method names</a>
```

Comment:
This list is out of date... what's the plan to keep these CDDL files in sync w/ the DID Spec registries (if we don't autogenerate some of the CDDL, it will be wrong; for example, the list of DID Methods is changing on a fairly regular basis.